sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -262,8 +262,14 @@ class SQLContext(@transient val sparkContext: SparkContext)
    *
    * @group userf
    */
-  def parquetFile(path: String): DataFrame =
-    DataFrame(this, parquet.ParquetRelation(path, Some(sparkContext.hadoopConfiguration), this))
+  @scala.annotation.varargs
+  def parquetFile(paths: String*): DataFrame =
```

Comment:
Are we actually ever going to do that for this function?  This makes it harder to do something like `parquetFile(listOfFiles: _*)` which I think is actually a common usecase.